sound/soc/intel/boards/sof_es8336.c

```diff
@@ -332,22 +310,13 @@ static int sof_es8336_quirk_cb(const struct dmi_system_id *id)
  * if the topology file is modified as well.
  */
 static const struct dmi_system_id sof_es8336_quirk_table[] = {
-	{
-		.callback = sof_es8336_quirk_cb,
-		.matches = {
-			DMI_MATCH(DMI_SYS_VENDOR, "IP3 tech"),
-			DMI_MATCH(DMI_BOARD_NAME, "WN1"),
-		},
-		.driver_data = (void *)(SOF_ES8336_SPEAKERS_EN_GPIO1_QUIRK)
-	},
 	{
 		.callback = sof_es8336_quirk_cb,
 		.matches = {
 			DMI_MATCH(DMI_SYS_VENDOR, "HUAWEI"),
 			DMI_MATCH(DMI_BOARD_NAME, "BOHB-WAX9-PCB-B2"),
 		},
-		.driver_data = (void *)(SOF_ES8336_HEADPHONE_GPIO |
-					SOC_ES8336_HEADSET_MIC1)
```

Comment:
Due to the above, we still need at least one quirk:
- a quirk to select if GPIO 0 is speaker or microphone;
- (maybe) a quirk to indicate if true means enabled or disabled for headphone (and maybe another for speaker if needed)